build_web_compilers/lib/src/dev_compiler_builder.dart

```diff
@@ -23,7 +24,10 @@ const jsSourceMapExtension = '.ddc.js.map';
 
 /// A builder which can output ddc modules!
 class DevCompilerBuilder implements Builder {
-  const DevCompilerBuilder();
+  final bool useKernel;
```

Comment:
probably not worth it right now, but I wonder whether it would help to simply do a DDK builder separate from the DDC builder instead (since so much of the code has special casing for the ddk case.)

Reply:
The advantage of this route is we can allow you to switch modes by changing the config option for the builder, which is a bit easier to do.